docs/install.rst

```diff
@@ -348,3 +348,38 @@ If creating a new environment doesn't work then it may be time to uninstall and
 Note that this means you will lose all the environments you have created.
 Please follow these `uninstall instructions`_ to uninstall conda.
 After uninstalling, you should be able to reinstall using the regular installation instructions.
+
+
+**Conda-forge is very slow in China, what can I do?**
+
+Conda-forge can be slow when accessed from China.
+In this case, it may be wise to use a Chinese mirror to install Pylake.
+Unfortunately, there is no mirror for `conda-forge`.
+This means that we cannot use the regular `conda-forge` installation instructions that were outlined above.
+
```

Comment:
I'd prefer slightly different wording:

Conda-forge can be slow when accessed from China. This can be resolved by using a Chinese mirror to install Pylake. Since there is no mirror for 'conda-forge', Pylake then has to be installed using pip, as outlined below.

Reply:
Yes, much better! Amended!